### PR TITLE
Update .gitignore

### PR DESCRIPTION
**Pull request by robinsuri**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@
 lll
 asdfsadf
```
Comment (githubapptesting) [inline .gitignore]: change this

Comment (githubapptesting) [inline .gitignore]: start a review

Comment (githubapptesting) [inline .gitignore]: aalskdj

Comment (githubapptesting) [inline .gitignore]: lkjlkjlkj

**Comment by robinsuri** [comment]
comment

**Comment by robinsuri** [comment]
gtm

**Pull request merged**